lib/ash_phoenix/form/form.ex

```diff
@@ -239,10 +239,10 @@ defmodule AshPhoenix.Form do
       Example:
       ```
       AshPhoenix.Form.for_create(..., transform_errors: fn
-        %{field: :field1} = error ->
+        _changeset, %{field: :field1} = error ->
```

Comment:
🤔 I think thats the form not the changeset, could you double check please?

Reply:
Running each argument through `IO.inspect`, I can confirm that the first argument is a changeset, and the second is an error.